docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller_test.go

```diff
@@ -0,0 +1,200 @@
+/*
+We can create the tests in a separate file, cronjob_controller_test.go.
+Keeping consistent with the Gingko usage in our autogenerated suite_test.go code, we’ll use Gingko to write our tests.
+*/
+
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// +kubebuilder:docs-gen:collapse=Apache License
+
+/*
+As usual, we start with necessary imports. We also define some utility variables.
+*/
+package controllers
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	batchv1 "k8s.io/api/batch/v1"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+
+	cronjobv1 "tutorial.kubebuilder.io/project/api/v1"
+)
+
+const (
+	CronjobName      string = "test-cronjob"
+	CronjobNamespace string = "test-cronjob-namespace"
+	JobName          string = "test-job"
+
+	timeout  = time.Second * 10
+	interval = time.Millisecond * 250
+)
+
+/*
+We also expose the GVK for our CronJob version, which we will use to manage ownership of Jobs that belong to our test CronJob.
+One way you can set this GVK is by defining it as a global var like so:
+*/
+var (
+	Kind string                  = reflect.TypeOf(cronjobv1.CronJob{}).Name()
```

Comment:
The only place that is using it is the next line .. so I am not sure if it would be required at all.

Reply:
`Kind` is only used in one place but I do like breaking `Kind` and `GVK` apart here--it makes the code more readable (rather than cramming this all into one line) and also maps nicely to the Kind and GVK concepts discussed in earlier documentation chapters like https://book.kubebuilder.io/cronjob-tutorial/gvks.html. What do you think?